TextField/TextField.jsx

```diff
@@ -78,10 +78,16 @@ class TextFieldInput extends MaterialComponent {
     if ("value" in props && this.state.showFloatingLabel) {
       className = [className, "mdc-text-field--upgraded"].join(" ");
     }
+    if (props.label && props.fullwidth) {
```

Comment:
not really sure about this

a. This will error even in production version

b. this might confuse devs as in what deleted the label

may be we can just leave a note in docs

Reply:
Would you rather delete the label and _not_ log the warning?

Reply:
To be honest, I'd rather log an error in every case, doesn't hurt production in my opinion but makes it a lot better for developer experience. Wouldn't remove the label though, so you'd at least see that there's a problem.

My first instinct would be the check the console to see if there are no errors, but it might not be the case for everyone. Might even throw an exception so that debuggers pick it up?

Reply:
Throwing an exception stops execution. Just leaves them at an empty screen

Reply:
Well, I know, but at least it doesn't fail "silently" and the user immediatly knows that something's wrong. That's the way Angular does it. Granted, they make the error quite a bit more beautiful but still.

I don't know if it's an ideal solution though, just the one that seems easiest to debug I guess?

Reply:
actually failing silently isn't a big issue as `mdc-web` doesn't prohibit this behaviour. Lets just console this and leave for the user if he still wants it and want to style it in his own way